src/RawData/storageController.ts

```diff
@@ -118,4 +123,6 @@ export const processRawDataFile = async (fileId: string) => {
         resolve(undefined);
```

Comment:
Should the "Waiting for parameters" loop above have some more specific logging information? Something like "processing parameter `x`" for each parameter function?

Reply:
There are a lot of parameters so I found that it causes too much noise in the logs.

Reply:
Cool, no worries